client/components/tinymce/plugins/media/drop-zone.jsx

```diff
@@ -107,18 +144,67 @@ export default React.createClass( {
 		if ( isSingleImage && ! MediaValidationStore.hasErrors( site.ID ) ) {
 			// For single image upload, insert into post content, blocking save
 			// until the image has finished upload
-			if ( selectedItems[ 0 ].transient ) {
+			let selectedItem = selectedItems[ 0 ]
```

Comment:
If you're not reassigning `selectedItem`, should be declared with `const` ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#variable-declarations)).